src/command-parser/parsers.ts

```diff
@@ -39,20 +42,41 @@ export const parseBIP32Path = (
   throw Error(Errors.InvalidPathError)
 }
 
-export const parseFileTypeMagic = (fileTypeMagic: string, path: string) => {
-  if (fileTypeMagic.startsWith('Payment')) {
-    return HwSigningType.Payment
-  }
-
-  if (fileTypeMagic.startsWith('StakePool')) { // TODO this string should be a symbolic constant, occurs twice
-    return HwSigningType.PoolCold
+export const parseFileTypeMagic = (fileTypeMagic: string, pathType: PathTypes): HwSigningType => {
+  // cardano-cli only distinguish 3 categories, payment, pool cold and stake keys
+  // to keep things simple, any other key is bundled into the "payment" category
+  // to keep things more generic
+  const fileTypeStartsWith = (label: PathLabel) => {
```

Comment:
```suggestion
  const checkFileTypeStartsWith = (label: PathLabel) => {
```

Reply:
Renamed in https://github.com/vacuumlabs/cardano-hw-cli/pull/98/commits/211c51dfb44e953b055ce8fa529aa843becb445f